specs/lfp-appeals-api-spec.json

```diff
@@ -0,0 +1,375 @@
+{
+  "openapi": "3.0.1",
```

Comment:
The `description` is missing for many parameters/fields/..etc. This would be useful for developers to understand the data that needs to be sent to the API and what they are receiving.